csuntechlab/iSTART
# STR-110 -- BE: Email about group is sent out on every log in, not when the user is assigned to the group

# Description

THIS TASK IS DONE WHEN:

Emails are no longer being sent on every log in, only when a user is being assigned to a group
  
# Testing
1. Log in as any user besides steve
2. Notice that no email is being sent
3. Rejoice
